keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ClusterDeployStrategy.kt

```diff
@@ -87,7 +97,8 @@ data class RedBlack(
 @JsonTypeName("highlander")
 object Highlander : ClusterDeployStrategy() {
   override fun toOrcaJobProperties() = mapOf(
-    "strategy" to "highlander"
+    "strategy" to "highlander",
+    "stageTimeoutMs" to Duration.ofMinutes(30).toMillis()
```

Comment:
This should probably be `DEFAULT_WAIT_FOR_INSTANCES_UP` as well

Reply:
OK, will fix.